ts/utils/payment.ts

```diff
@@ -307,3 +317,21 @@ export const getPaymentOutcomeCodeDescription = (
   }
   return none;
 };
+
+export const getPickPaymentMethodDescription = (
+  bancomatOrCreditCard:
+    | CreditCardPaymentMethod
+    | PrivativePaymentMethod
+    | BancomatPaymentMethod
+) => {
+  const translatedExpireDate = getTranslatedShortNumericMonthYear(
+    bancomatOrCreditCard.info.expireYear,
+    bancomatOrCreditCard.info.expireMonth
+  );
+  return translatedExpireDate
+    ? I18n.t("wallet.payWith.pickPaymentMethod.description", {
+        firstElement: translatedExpireDate,
+        secondElement: bancomatOrCreditCard.info.holder
```

Comment:
It could happen that `holder` is `undefined`
What do you think to add a fallback?

```typescript
export const getPickPaymentMethodDescription = (
  bancomatOrCreditCard:
    | CreditCardPaymentMethod
    | PrivativePaymentMethod
    | BancomatPaymentMethod,
  defaultHolder: string = ""
) => {
  const translatedExpireDate = getTranslatedShortNumericMonthYear(
    bancomatOrCreditCard.info.expireYear,
    bancomatOrCreditCard.info.expireMonth
  );
  return translatedExpireDate
    ? I18n.t("wallet.payWith.pickPaymentMethod.description", {
        firstElement: translatedExpireDate,
        secondElement: bancomatOrCreditCard.info.holder ?? defaultHolder
      })
    : fromNullable(bancomatOrCreditCard.info.holder).getOrElse("");
};
```

```typescript
case "Bancomat":
      return {
        logo: pagoBancomatLogo,
        title: paymentMethod.caption,
        description: getPickPaymentMethodDescription(
          paymentMethod,
          nameSurname
        ),
        bottomSheetTitle: unacceptedBottomSheetTitle(),
        bottomSheetBody: unacceptedBottomSheetBody()
      };
```

| before  | after |
| ------------- | ------------- |
![before](https://user-images.githubusercontent.com/822471/112823220-4d7eb180-9089-11eb-8f35-d720d72f9db4.jpg) | ![after](https://user-images.githubusercontent.com/822471/112823216-4c4d8480-9089-11eb-9a53-b36710c545ad.jpg)

Reply:
Good idea, updated in [90c3750](https://github.com/pagopa/io-app/pull/2926/commits/90c3750ac0d1e9ca87b57ea243198ba25bc08035)